portfolio/src/main/java/com/google/sps/servlets/UserComment.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import org.javatuples.Pair;
+
+public class UserComment {
+  // wrapped comment entity that contains the comment message and image url
+  private Pair<String, String> commentEntity;
```

Comment:
Why pair if you have a separate class already?
Can simply have:
private String commentMsg;
private String imageUrl;